docs/guides/Package-authoring-best-practices.md

```diff
@@ -0,0 +1,177 @@
+
+### Authors
+
+✔️ CONSIDER including the current NuGet.org owners for consistency across NuGet clients.
+> Visual Studio currently displayed `Authors` on the Browse tab. It is also in the criteria for ID prefix reservation.
+
+### Description
+
+✔️ DO include a short description for your package that describes what it's for.
+> Package descriptions are one of the most prominent fields surfaced in NuGet search and will likely be the first thing a potential consumers looks at to determine if a package is right for them.
+
+### Copyright
+
+✔️ CONSIDER copyrighting your package with "Copyright (c) <name/company\> <year\>."
+>A copyright notice essentially indicates that your work cannot be copied without your permission. Including, a copyright notice in your package is easy and won't do any harm!
+
+Example: Copyright (c) Contoso 2020
+
+### Licensing
+
+✔️ DO specify a valid license for your package.
+> [!IMPORTANT]
+> A project without a license defaults to [exclusive copyright](https://choosealicense.com/no-permission/), making it legally impossible for other people to use.
+
+✔️ CONSIDER using a license expression (SPDX identifier).
+> License expressions are surfaced the most clearly to package consumers and make it more obvious if the license has changed with a new release.
+
+✔️ CONSIDER specifying an MIT license.
+> If you want your package as to be usable by as many consumers as possible, the MIT license is a very well known non-restrictive license.
+
+❌ DO NOT use the deprecated `LicenseUrl` metadataproperty.
+> This presents legal ambiguity as license changes at the URL will retroactively change the displayed license for previous package versions.
+
+### Project URL
+
+✔️ CONSIDER including a link to an associated project or company website.
+> Company website links, even if not specific to the package/project, still improve consumer trust.
+
+❌ DO NOT put a repository URL (GitHub, BitBucket, etc.) in the `Project URL` field, as consumers will expect to find it in the `Repository URL` field.
+
+### Icon
+
+✔️ CONSIDER including an icon with your package to help visually differentiate it. It's a relatively small addition that can improve perception of package quality.
+> Icons can be specific to individual packages or be a brand logo.
+
+✔️ DO use an image that is 128x128 and has a transparent background (PNG) for best viewing results.
+> NuGet will automatically scale your image to the client it is being displayed on.
+
+❌ DO NOT use the deprecated `IconUrl` metadata property.
+
+### Repository Type and URL
+
+✔️ CONSIDER including a repository type and URL to make it easy for package consumers to view sourecode and package documentation (only if public).
+
+✔️ CONSIDER setting up [Source Link](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/sourcelink) to add source control metadata to your assemblies and NuGet package.
```

Comment:
I think we should expand on this:

https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/nuget#symbol-packages

And this doc?

https://docs.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg

Reply:
Yes, I think calling out "snupkg"/symbols packages is important. Source link is one path but it's not the only one and if a user can't user Source Link for some reason, consider still publishing a snupkg with, say, embedded sources.

And what about the snupkg vs. PDB in the .nupkg discussion. Any recommendations around that?

Reply:
Our official recommendation has been to put PDBs in the .snupkg (to reduce .nupkg size), but this has been controversial as many argue that embedded PDBs provide a simpler experience.

Reply:
What's the experience for including PDBs in the package vs in a .snupkg?

I always just add this:
```
<PropertyGroup>
    <IncludeSymbols>true</IncludeSymbols>
    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
</PropertyGroup>
```

Reply:
If we're talking snupkgs here we need to call out the portable pdb caveat.

Reply:
I've heard the argument of snupkg vs embedded symbols so many times that I don't think there is an official recommendation. Let's just put in both scenarios for now until that's clearer by the .NET team / etc?